entity-framework/core/providers/sql-server/functions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Function Mappings - Microsoft SQL Server Database Provider - EF Core
+description: Function Mappings of the Microsoft SQL Server database provider
+author: bricelam
+ms.date: 10/07/2020
+uid: core/providers/sql-server/functions
+---
+# Function Mappings of the Microsoft SQL Server Provider
+
+This page shows which .NET members are translated into which SQL functions when using the SQL Server provider.
+
+## Binary functions
+
+.NET                         | SQL                           | Added in
```

Comment:
```suggestion
C#                           | SQL                           | Added in
```

Reply:
Or maybe all of the below is OK on VB too :sweat_smile:

Reply:
I'll stick with *.NET*. They're not exactly code listings...